client/src/components/Modal/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { ModalController, ModalData } from '../../types/modal';
+import {
+  BottomRightButton,
+  Bottom,
+  Background,
+  Wrapper,
+  Title,
+  SubTitle,
+} from './style';
+
+function Modal({
+  props,
```

Comment:
```suggestion
  props: { title, subTitle, middleContent, bottomRightButton },
```
바로 구조분해쌉가능

MDN 구조 분해 할당의 중첩된 객체 및 배열의 구조 분해 항목을 한번 보면 좋을 것 같음!
https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#%EC%A4%91%EC%B2%A9%EB%90%9C_%EA%B0%9D%EC%B2%B4_%EB%B0%8F_%EB%B0%B0%EC%97%B4%EC%9D%98_%EA%B5%AC%EC%A1%B0_%EB%B6%84%ED%95%B4